package.json

```diff
@@ -1,28 +1,39 @@
 {
   "name": "universal-search-content",
-  "version": "1.0.0",
-  "description": "the iframe that we put inside the autocomplete dropdown",
+  "version": "0.0.1",
+  "description": "",
   "main": "index.js",
   "scripts": {
+    "build": "./node_modules/.bin/webpack --optimize-minimize --optimize-dedupe",
+    "start": "./node_modules/.bin/webpack-dev-server --content-base dist/",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/mozilla/universal-search-content"
+  "author": "Mozilla",
+  "license": "MPL 2.0",
```

Comment:
Should be "MPL-2.0", if we want to be SPDX compliant (or newer npm's will whine).